docs/source/chapt_opt/tutorial/parameter_est.rst

```diff
@@ -171,7 +171,7 @@ objective function calculation.
 20. For larger sample sets, **Generate Samples** has an option to load
     from a CSV file. The CSV file must be saved as "CSV (MS-DOS)" as the file type, as follows:
 
-.. figure:: ../figs/optim_samplevars_csv.png
```

Comment:
It looks like this is the only place this image is referenced.  If it's getting replaced by a new file (optim_csv_samples_format.png) seems like the old one should be removed, right?